Unit-01/03-templating/app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, render_template, url_for, request
+
+app = Flask(__name__)
+
+@app.route("/person/<name>/<age>")
+def show(name, age):
+	return render_template('person-name-age-show.html', name = name, age = age)
```

Comment:
nitpicking here, but it's usually considered more "Pythonic" to not have a space between kwarg operator, e.g. `(name=name, age=age)`